scarb/src/ops/proc_macro_server/json_rpc.rs

```diff
@@ -0,0 +1,12 @@
+use serde::Serialize;
+
+#[derive(Serialize)]
+pub struct ErrResponse {
```

Comment:
this should probably reside in proc-macros-server-api crate. I made a related comment about this in that PR: https://github.com/software-mansion/scarb/pull/1678#discussion_r1816600767